receiver/prometheusreceiver/internal/metricfamily.go

```diff
@@ -37,9 +37,12 @@ type MetricFamily interface {
 }
 
 var (
-	k8SLocation    = "k8s_location"
-	k8SClusterName = "k8s_cluster_name"
-	k8SNodeName    = "k8s_node_name"
+	k8SKeyLocation      = "k8s_location"
+	k8SKeyClusterName   = "k8s_cluster_name"
+	k8SKeyNodeName      = "k8s_node_name"
+	k8SKeyContainerName = "container_name"
```

Comment:
why are these names not consistent? existing names have 'k8s_' prefix while new ones don't.

Reply:
This has been removed.